db/keyval/redis/bytes_broker_impl.go

```diff
@@ -220,16 +229,24 @@ func (it *bytesKeyIterator) GetNext() (key string, rev int64, lastReceived bool)
 	return key, 0, false
 }
 
+func (it *bytesKeyValIterator) Close() error {
```

Comment:
Yes, I mean Close() method comment. There should be also "DISCLAIMER" also in method that returns iterator...